src/features/landing/components/Possibilities/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,61 @@
+'use client';
+
+import { Box, Grid, Typography } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+}
+
+function CardItem({ item }: CardItemInterface) {
```

Comment:
you have 2 duplicated cardItem components? need rework this 
https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself